client/src/App.tsx

```diff
@@ -1,24 +1,58 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { RefObject, useState } from "react";
+import Header from "./components/header/Header";
+import Main from "./components/Main/Main";
+import NewNote from "./components/NewNote";
 
+export interface NotesProps {
+  id: number;
+  title: string | null;
+  body?: string | null;
+  createdAt: string;
+}
+
+// TODO: Refactor
 function App() {
+  const [notes, setNotes] = useState<NotesProps[]>([
+    { id: 1, title: "Note One", body: "Lorem Ipsum", createdAt: "Jun 01 2021" },
+    { id: 2, title: "Note One", body: "Lorem Ipsum", createdAt: "Jun 01 2021" },
+    { id: 3, title: "Note One", body: "Lorem Ipsum", createdAt: "Jun 01 2021" },
+    { id: 4, title: "Note One", body: "Lorem Ipsum", createdAt: "Jun 01 2021" },
+  ]);
+  const [isCreateNote, setIsCreateNote] = useState(false);
+
+  const showCreateNote = () => setIsCreateNote(true);
+  const closeCreateNote = (e: any) => {
+    e.preventDefault();
+    setIsCreateNote(false);
+  };
+  const storeNote = (
+    e: any,
+    title: RefObject<HTMLInputElement>,
+    body?: RefObject<HTMLTextAreaElement>
+  ) => {
+    e.preventDefault();
+    const id = notes.length;
+    const newNote = {
+      id,
+      title: title.current!.value,
+      body: body?.current!.value,
+      createdAt: new Date().toString(),
+    };
+    setNotes([...notes, newNote]);
+    setIsCreateNote(false);
+    title.current!.value = "";
+    if (body) body.current!.value = "";
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/zerexei/zoop-note/pullRequest?prid=7527662)
```suggestion
    if (body) {body.current!.value = "";}
```